otter/integration/tests/test_convergence.py

```diff
@@ -351,6 +351,74 @@ def test_scale_over_group_max_after_metadata_removal(self):
             )
         )
 
+    @tag("CATC-019")
+    def test_scale_over_lb_limit(self):
+        """
+        CATC-019-a: Validate that when a group attempts to scale over the
+        load balancer limit the group enters an error state. It is expected
+        that the active capacity will match the maximum allowed on the load
+        balancer and the pending capacity will become zero once the group is
+        in ERROR state.
+
+        1. Creates a scaling group with a load balancer and max servers
+           greater than the load balancer max. (Assume LB_max == 25)
+        2. Scale up to a desired capacity greater than the LB_max
+        3. Assert that the scaling group goes into error state, that the
+           active count is equal to LB_max, and that no servers are pending.
+
+        """
+        LB_max = 25
+        group_max = 30
+
+        rcs = TestResources()
+        self.helper = TestHelper(self, num_clbs=1)
+
+        # def create_clb_first():
+        #     self.clb = CloudLoadBalancer(pool=self.helper.pool)
+        #     self.helper.clbs = [self.clb]
+        #     return (
+        #         self.identity.authenticate_user(
+        #             rcs,
+        #             resources={
+        #                 "otter": (otter_key, otter_url),
+        #                 "nova": (nova_key,),
+        #                 "loadbalancers": (clb_key,)
+        #             },
+        #             region=region,
+        #         ).addCallback(self.clb.start, self)
+        #         .addCallback(self.clb.wait_for_state, "ACTIVE", 600)
+        #     )
+
+        # def then_test(_):
+        self.scaling_group = self.helper.create_group(
+            image_ref=image_ref, flavor_ref=flavor_ref, min_entities=1,
+            max_entities=group_max)
+
+        self.scale_up_to_group_max = ScalingPolicy(
+            set_to=LB_max,
```

Comment:
Shouldn't this be set to something like `LB_max + 1` if we're to test behavior of it going over CLB maximums?